SingularityUI/app/components/tasks/Columns.jsx

```diff
@@ -116,26 +116,26 @@ export const Memory = (
 );
 
 export const ActiveActions = (onTaskKill) => {
-  return <Column
-    label=""
-    id="actions"
-    key="actions"
-    className="actions-column"
-    cellData={
-      (rowData) => rowData
-    }
-    cellRender={(cellData) => {
+  return (
```

Comment:
is the return here part of our style guide? i dont necessarily have an issue with it, but am curious if it's there for an explicit reason